src/pages/HomePage.module.css

```
@@ -49,35 +82,62 @@
   flex-shrink: 0;
   justify-content: center;
 
-  margin-top: 100px;
+  margin-top: 90px;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)